### PR TITLE
feature: add quick works support for disk fill 

### PR DESCRIPTION
**Pull request by sequarius**
### Describe what this PR does / why we need it
call `fallocate` to fill disk space without realy io, when the environment filesystem is common linux filesystem(ext4, xfs, btrfs and ocfs2 etc.)

### Does this pull request fix one issue?
see #143

### CODE REVIEW

**Comment by jievince** [comment]
I think it's better to write in this way.
```
func startFill(mountPoint, size string) {
	ctx := context.Background()
	if mountPoint == "" {
		bin.PrintErrAndExit("mount-point flag is empty")
	}
	dataFile := path.Join(mountPoint, fillDataFile)
	if !exec.IsCommandAvailable("fallocate") {
		bin.PrintErrAndExit("fallocate command not exist")
	}
	response := channel.Run(ctx, "fallocate", fmt.Sprintf(`-l %sM %s`, size, dataFile))
	if !response.Success {
		stopFill(mountPoint)
		bin.PrintErrAndExit(response.Err)
	}
	bin.PrintOutputAndExit(response.Result.(string))
}
```

**Comment by sequarius** [comment]
OK, I Change to use `exec.IsCommandAvailable("fallocate")` to find if the `fallcocate` can be invoke at this environment, But I think the `dd` implement should be keep to ensure this expriment can run on any other else filesystem even some limit be found in #143

**Comment by xcaspar** [comment]
@sequarius Please check the following error messages:
```
exec/os/bin/filldisk/filldisk.go:58:2: undefined: response
exec/os/bin/filldisk/filldisk.go:59:6: undefined: response
exec/os/bin/filldisk/filldisk.go:61:23: undefined: response
exec/os/bin/filldisk/filldisk.go:63:2: undefined: response
exec/os/bin/filldisk/filldisk.go:65:6: undefined: response
exec/os/bin/filldisk/filldisk.go:67:23: undefined: response
exec/os/bin/filldisk/filldisk.go:69:25: undefined: response
Makefile:106: recipe for target 'build_filldisk' failed
make: *** [build_filldisk] Error 2
```

File: exec/os/bin/filldisk/filldisk.go [hunk 1]
```diff
@@ -44,7 +44,18 @@ func startFill(mountPoint, size string) {
...
+	if exec.IsCommandAvailable("fallocate") {
+		response := channel.Run(ctx, "fallocate", fmt.Sprintf(`-l %sM %s`, size, dataFile))
+		if !response.Success {
+			stopFill(mountPoint)
+			bin.PrintErrAndExit(response.Err)
+		}
+		bin.PrintOutputAndExit(response.Result.(string))
+	}
+	
+	response = channel.Run(ctx, "dd", fmt.Sprintf(`if=/dev/zero of=%s bs=1b count=1 iflag=fullblock`, dataFile))
```
Comment (jievince) [inline exec/os/bin/filldisk/filldisk.go]: Should be `response :=`

**Comment by jievince** [comment]
See my comment [here](https://github.com/chaosblade-io/chaosblade/pull/146/files#r308040604)

**Comment by xcaspar** [comment]
@sequarius After you fix the above problem, you can use the rebase command to merge your commits into one commit.

**Comment by sequarius** [comment]
@xcaspar  oh, stupid error，had been shot.

**Comment by xcaspar** [comment]
@sequarius Thanks 👍

**Comment by itcai** [comment]
good job

**Pull request merged**